app/src/main/java/com/team7/cmput301/android/theirisproject/activity/ProblemSlideshowActivity.java

```diff
@@ -0,0 +1,34 @@
+package com.team7.cmput301.android.theirisproject.activity;
+
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+
+import com.team7.cmput301.android.theirisproject.ImageSlideAdapter;
+import com.team7.cmput301.android.theirisproject.R;
+import com.team7.cmput301.android.theirisproject.controller.ProblemController;
+import com.team7.cmput301.android.theirisproject.controller.RecordController;
+
+public class ProblemSlideshowActivity extends AppCompatActivity {
+
+    private RecordController controller;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_problem_slideshow);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        controller = new RecordController(getIntent());
```

Comment:
Slideshow should have a list of records so maybe RecordListController is more suitable in this situation

Reply:
Suggestion has been implemented and comments have been added